website/content/docs/integrations/vault/acl.mdx

```diff
@@ -763,25 +763,62 @@ $ VAULT_TOKEN=s.H39hfS7eHSbb1GpkdzOQLTmz.fvuLy nomad job run vault.nomad
 
 Migrating from the legacy (pre-1.7) workflow where workloads use the agent's
 Vault token requires configuration on your Vault cluster and your Nomad server
-agents. It does not require updating your running Nomad jobs unless you wish to
-specify a non-default role. To migrate:
+agents.
+
+Once the migration is fully complete, Nomad server will no longer have access
+to Vault, as it was required in the deprecated legacy workflow. This also means
+that they will no longer be able to fulfill some of their responsibilities from
+the legacy workflow, such as generating and revoking Vault ACL tokens.
+
+Before removing Vault connectivity configuration from Nomad servers, you must
+make sure the rest of the cluster is ready to support workload identities for
+Vault.
+
+You can run the [`nomad setup vault -check`][nomad_cli_setup_vault] command to
+verify what changes are still necessary.
```

Comment:
Should probably be the same paragraph, other wise I'm left wondering what the last sentence is pertaining to. (initially thought it was about upgrading)